autoload/denops/_internal/server/proc.vim

```diff
@@ -84,18 +85,36 @@ function! s:on_stdout(store, data) abort
   call denops#_internal#event#emit(printf('DenopsSystemProcessListen:%s', l:addr))
 endfunction
 
-function! s:on_stderr(data) abort
+function! s:on_stderr(store, data) abort
   echohl ErrorMsg
   for l:line in split(a:data, '\n')
     echomsg printf('[denops] %s', substitute(l:line, '\t', '    ', 'g'))
   endfor
   echohl None
+  if a:store.stderr->add(a:data)->len() > s:LAST_ERROR_LINES
+    eval a:store.stderr->remove(0, -(s:LAST_ERROR_LINES + 1))
+  endif
```

Comment:
If the error is reported through `stderr`, I think we can catch that error on Deno side instead of Vim side. What do you think?

Reply:
Ah, got it by https://github.com/vim-denops/denops.vim/pull/403#discussion_r1706267525. Nvm